_dprhtml/features/settings-dialog/main-pane.html

```diff
@@ -154,13 +154,13 @@ <h6 class="modal-title" id="exampleModalLongTitle">Digital Pali Reader Preferenc
                 <div class="settings-pane-tab-pane-section-header">Text formatting</div>
 
                 <label for="colfont" style="white-space: nowrap;">Font Family</label>
-                <input class="ml-1" id="colfont" name="colfont" type="text" maxlength="16" style="justify-self: stretch;"/>
+                <input class="ml-1" id="colfont" name="colfont" type="text" maxlength="16" style="justify-self: stretch;" data-bind="value: fontFamily"/>
 
                 <label for="colsize" style="white-space: nowrap;">Text Size</label>
-                <input class="ml-1" id="colsize" name="colsize" type="text" maxlength="16" style="justify-self: stretch;"/>
+                <input class="ml-1" id="colsize" name="colsize" type="text" maxlength="16" style="justify-self: stretch;" data-bind="value: textSize"/>
```

Comment:
could we retain the names consistent with the other places. i.e. instead of "value: textSize", we use "value: colsize"